project.yml

```diff
@@ -14,7 +14,14 @@ fileGroups:
   - install-git-hook.sh
   - renovate.json5
 
+options:
+  localPackagesGroup: Modules
+
 packages:
+  SharedUI:
+    path: Modules/SharedUI
+  WRCore:
+    path: Modules/WRCore
   shared:
```

Comment:
Isn't this implicit included by Modules/WRCore?

Reply:
Yes but it could be that you want to directly access shared so I added it here too.